src/components/studies/scheduler/ConfigureBurstTab.tsx

```diff
@@ -129,41 +212,59 @@ const ConfigureBurstTab: FunctionComponent<ConfigureBurstTabProps> = ({
     onUpdate(schedule)
   }
 
+  const displayBurstInfoText =
+    schedule && hasBursts && (burstNumber || 0) > 0 && (burstFrequency || 0) > 0
   return (
     <div className={classes.root}>
-      <MTBHeadingH1>Multiple scheduling of the same events</MTBHeadingH1>
-      <p>
-        Ecological Momentary Assessment (EMA) provides a way to repeatedly
-        sample participant over time to better capture their ehaviors and
-        experiences in real time, also referred to as Study Bursts.
-      </p>
-      <p>
-        Example: Participants will take a session of 4 assessments everyday for
-        a week. They redo this burst every 3 months for the next two years.
+      <MTBHeadingH1 className={classes.burstDesignHeader}>
+        Burst Design
+      </MTBHeadingH1>
+      <p className={classes.paragraph}>
+        A burst design involves repeating multiple study sessions that are
+        spaced out over time at regular intervals with long breaks. This is
+        intended to maximize longitudinal participation with minimal burden.
       </p>
-      <MTBHeadingH2>Will your study include an EMA/Study Burst? </MTBHeadingH2>
-      <ToggleButtonGroup
-        value={hasBursts}
-        exclusive={true}
-        onChange={(e: React.MouseEvent<HTMLElement>, val: boolean) => {
-          if (val !== null) {
-            setHasBursts(val)
-          }
-        }}
-        aria-label="study includes bursts">
-        <ToggleButton value={false} aria-label="no">
-          No
-        </ToggleButton>
-        <ToggleButton value={true} aria-label="yes">
-          Yes
-        </ToggleButton>
-      </ToggleButtonGroup>
+      <Box className={classes.burstSummaryContainer}>
+        <MTBHeadingH2 className={classes.burstDesignHeading}>
+          Will your study include a Burst Design?
+        </MTBHeadingH2>
+        <ToggleButtonGroup
+          value={hasBursts}
+          exclusive={true}
+          classes={{root: classes.toggleButtonRoot}}
+          onChange={(e: React.MouseEvent<HTMLElement>, val: boolean) => {
+            if (val !== null) {
+              setHasBursts(val)
+            }
+          }}
+          aria-label="study includes bursts">
+          <ToggleButton
+            value={true}
+            aria-label="yes"
+            className={classes.buttons}>
+            Yes
+          </ToggleButton>
+          <ToggleButton
+            value={false}
+            aria-label="no"
+            style={{marginLeft: '16px', border: '1px solid black'}}
```

Comment:
can you use classes with css selector (:first-child, ":last-child etc to avid the style here and include it in the class instead?)